csvapi/profileview.py

```diff
@@ -17,23 +17,26 @@
 import json
 
 from csvapi.setup_logger import logger
+
+
 class ProfileView(MethodView):
 
     def get_dataframe(self, db_info, dtype=None):
         dsn = 'file:{}?immutable=1'.format(db_info['db_path'])
-        conn = sqlite3.connect(dsn)
+        conn = sqlite3.connect(dsn, uri=True)
         sql = 'SELECT * FROM [{}]'.format(db_info['table_name'])
         try:
             df = pd.read_sql_query(sql, con=conn, dtype=dtype)
-        except:
+        # TODO: check if correct exception type
```

Comment:
Could you check that it was the correct exception type?

Reply:
Don't know how 🤷 Anyway this code does not look very clean and tested, introducing a breakage should not be the end of the world.

Reply:
I'm ok with that. Tagging @geoffreyaldebert here so that he can be aware of this.